README.md

```diff
@@ -80,6 +80,10 @@ Pass an input element into a new instance of `Ca11y`, and optionally pass in pro
     'Thirtieth',
     'Thirty First'
   ],
+  toggle: {
+    html: '📅',
```

Comment:
Hm. Good question. Yeah, I think I want to leave the SVGs as the default, but have defaults imported modularly, rather than bundled with everything. They're open source, from the google material icon set.